code/datums/martial/boxing.dm

```diff
@@ -281,6 +296,66 @@
 	honorable_boxer = FALSE
 	boxing_traits = list(TRAIT_BOXING_READY, TRAIT_STRENGTH, TRAIT_STIMMED)
 
+/// Hunter Boxing: for the uncaring, completely deranged one-spacer ecological disaster.
+/// The honor check accepts boxing ready targets, OR various biotypes as valid targets. Uses a special crit effect rather than the standard one (against monsters).
+/// I guess technically, this allows for lethal boxing. If you want.
+/datum/martial_art/boxing/hunter
+	name = "Hunter Boxing"
+	id = MARTIALART_HUNTER_BOXING
+	pacifist_style = FALSE
+	default_damage_type = BRUTE
+	boxing_traits = list(TRAIT_BOXING_READY)
+	/// The mobs we are looking for to pass the honor check
+	var/honorable_mob_biotypes = MOB_BEAST | MOB_SPECIAL | MOB_PLANT | MOB_BUG
+
+/datum/martial_art/boxing/hunter/honor_check(mob/living/possible_boxer)
+	// Unlike normal, this isn't an early return, because we're looking for MOB_HUMANOID as an early return. We DO want to return true for any instance where our target is boxing ready.
```

Comment:
This comment might need rewording because what's following it is still an early return, so I am not quite sure what it is saying.